app/views/spree/admin/products/bulk_edit/_products_head.html.haml

```diff
@@ -23,7 +23,7 @@
         = t(:expand_all)
     %th.producer{ 'ng-show' => 'columns.producer.visible' }=t('admin.producer')
     %th.sku{ 'ng-show' => 'columns.sku.visible' }=t('admin.sku')
-    %th.name{ 'ng-show' => 'columns.name.visible' }=t('admin.name')
+    %th.name{ 'ng-show' => 'columns.name.visible' }=t('admin.products.bulk_edit.name')
```

Comment:
did you try using `.name` as per Rails lazy lookup? See 4.1.4 in http://guides.rubyonrails.org/i18n.html. If possible, use the namespace in the other copies below :pray:

Reply:
hop! there it is